test/integration/controller/jobs/mpijob/mpijob_controller_test.go

```diff
@@ -547,7 +547,7 @@ var _ = ginkgo.Describe("Job controller for workloads when only jobs with queue
 		}, util.Timeout, util.Interval).Should(gomega.Succeed())
 	})
 
-	ginkgo.It("Should suspend a job if the parent workload does not exist", func() {
+	ginkgo.It("Should suspend a job if the parent's workload does not exist or is not admitted", func() {
```

Comment:
Can you add a similar test to this with a RayCluster owned by a RayJob?

Reply:
done